CHANGES.rst

```diff
@@ -244,6 +244,10 @@ astropy.coordinates
 astropy.cosmology
 ^^^^^^^^^^^^^^^^^
 
+- Docs for Onu method on cosmological objects no longer
```

Comment:
Could you please remove the entry? Docs only changes doesn't need to be mentioned in the changlog.